cmd/tester/main.go

```diff
@@ -126,12 +129,22 @@ func main() {
 
 	result, errs := webhook.ValidatePolicy(ctx, ns, ref, *cip, remoteOpts...)
 	errStrings := []string{}
+	warningStrings := []string{}
 	for _, err := range errs {
+		var fe *apis.FieldError
+		if errors.As(err, &fe) {
+			if fe.Level == apis.WarningLevel {
```

Comment:
`apis.FieldError` can contain a mix of errors and warnings.

We want to partition the errors and warnings with this filter method: https://github.com/knative/pkg/blob/a6afcab5bd78047384d858c913826af80dd7dcd8/apis/field_error.go#L168-L172

So `fe.Filter(apis.WarningLevel)` goes into `warningStrings`, and `fe.Filter(apis.ErrorLevel)` goes into `errStrings`